packages/api/src/lib/comments-api.ts

```diff
@@ -261,3 +266,23 @@ export async function deleteBlogPost(postId: string) {
     { headers: getAuthHeaders() }
   );
 }
+
+export async function uploadCdnImage(imageBase64: string): Promise<object> {
+  const response = await axios.post("/api/blog/upload-cdn-image", {
+    image: imageBase64,
+  });
+  if (response.data) {
```

Comment:
this code will not be reached. If you want to check for errors, you have to wrap the axios call in a try-catch block